.ci/gcb-contributor-membership-checker.yml

```diff
@@ -68,6 +68,7 @@ steps:
       - "membership-checker"
       - $_PR_NUMBER
       - $COMMIT_SHA
+      # The following are unused and should be safe to remove at least 2 weeks after this comment was added.
       - $BRANCH_NAME
       - $_HEAD_REPO_URL
       - $_HEAD_BRANCH
```

Comment:
Should we remove these now since it should be safe as we have `Args: cobra.RangeArgs(2, 6)` in the checker and change  it  to `Args: cobra.ExactArgs(2)` a few weeks after we make this change? The soak time is needed after we remove the unused args in the configuration file and before we change the check to accept exact two arguments, right?

Reply:
I think we need the soak time now because people with open PRs will _immediately_ get the changes to the cloudbuild.yml (but will still have the old magician code that requires all 6 args.) In two weeks it should be safe to remove these from cloudbuild since the new magician code allows either 2 or 6 - and at the same time we can update the magician code to only allow 2, which should be fine since people getting that version of the magician code will also have the cloudbuild.yml that only supplies 2

Reply:
ah, I'm thinking it's the opposite - open PRs should get old cloudbuild.yml for both membership and community checker but updated magician code since we check in what's in the main for both .ci/ and tools/ folder at https://github.com/GoogleCloudPlatform/magic-modules/blob/main/.ci/gcb-contributor-membership-checker.yml#L33. But I could be wrong

Reply:
hmm. No, I think you're right - cloudbuild.yml must come from the PR branch. With the token split, the issue was that the cloudbuild.yml was still supplying the old token, but the code expected the new token already. Thanks for catching this!

Reply:
fixed in f53f493fa